packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -385,6 +385,7 @@ export class DefaultPageBlockComponent
           </div>
           ${childrenContainer}
         </div>
+        <affine-drag-indicator></affine-drag-indicator>
```

Comment:
Querying this static component using DOM API doesn't look very right. This may also break in the edgeless mode. Is there a way for us to use slot-based events?

Reply:
> Slot-based events are a type of event that is associated with a specific time and location. These events are often used in calendar or scheduling applications and are designed to be scheduled or rescheduled as needed. The "slot" refers to the specific time and location that the event is scheduled to take place. This allows users to easily view and manage their schedule and see when and where events are taking place.

Does slot-based events mean as above? I think I get the point, I'll try this